library/envirophat/bmp280.py

```diff
@@ -153,15 +153,15 @@ def pressure(self):
         """Return the current air pressure."""
 
         self.update()
-        return self._pressure
+        return self._pressure / 10.0
 
     def altitude(self, qnh=QNH):
         """Return the current approximate altitude.
 
         :param qnh: Your local value for atmospheric pressure adjusted to sea level.
 
         """
-        return 44330.0 * (1.0 - pow(self.pressure() / (qnh*100), (1.0/5.255))) # Calculate altitute from pressure & qnh
+        return 44330.0 * (1.0 - pow((self.pressure() * 10) / (qnh*100), (1.0/5.255))) # Calculate altitute from pressure & qnh
```

Comment:
I think that `self.pressure() * 10` should be `self.pressure() * 100`.

Reply:
This could also be written:
  `return 44330.0 * (1.0 - pow((self.pressure() / qnh), (1.0 / 5.255)))`
as `qnh` is also in hectopascals, so the two values don't need to be converted for the math.